test/pytest/test_bridge.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import asyncio
+import json
+import sys
+import unittest
+
+from typing import Any, Dict, Tuple
+
+import systemd_ctypes
+from cockpit.router import Router
+
+asyncio.set_event_loop_policy(systemd_ctypes.EventLoopPolicy())
+
+
+class MockTransport(asyncio.Transport):
+    queue: asyncio.Queue[Tuple[str, bytes]]
+
+    def send_json(self, _channel: str, **kwargs) -> None:
+        self.send_data(_channel, json.dumps(kwargs).encode('ascii'))
```

Comment:
Totally unimportant style question: `_channel` is not an unused argument, why the _ prefix?

Reply:
Same as the reason we do it in protocol.py: this `_channel` is the channel in the sense of the first line of the message (ie: which is the empty string for control messages).  It needs to have a different name than `channel` which often appears within those messages.

Like, considering this message:

```
51

{"command":"open","channel":"1","payload":"echo"}
```

Then `_channel` is `''` and `channel` is `'1'`.